lib/discovery-indexer/reader/purlxml_model.rb

```diff
@@ -63,12 +63,28 @@ class PurlxmlModel
       #  ["pc0065_b08_f10_i031.txt","pc0065_b08_f10_i032.txt"]
       attr_accessor :file_ids
 
+      # @!attribute [rw] sw_file_ids
+      # @return [Array] a list of the file ids and corresponding druids with %2F
+      #  separator from the file and externalFile tags in the content_metadata
+      #  specifically for Searchworks
+      # @example
+      #  ["xx888yy9999%2Fpc0065_b08_f10_i031.pdf","aa111bb2222%2Fpc0065_b08_f10_i032.txt"]
+      attr_accessor :sw_file_ids
+
```

Comment:
yes, you are right.  I think Sandi Metz definitely had an influence on me - I was trying to make sure I had everything I needed...